.gitignore

```diff
@@ -160,3 +163,11 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+# Ruff
+.ruff_cache/
+
+# Others
+*inspo*
+wip-file*
+__main__.py
```

Comment:
This will bring trouble in the future when we add the entrypoint for the package